Content.Shared/Speech/Components/MeleeSpeechComponent.cs

```diff
@@ -51,3 +70,13 @@ public MeleeSpeechBattlecryChangedMessage(string battlecry)
         Battlecry = battlecry;
     }
 }
+
+public sealed class MeleeSpeechConfigureActionMessage : InstantActionEvent
+{
+    public EntityUid User { get; }
+
+    public MeleeSpeechConfigureActionMessage(EntityUid who)
+    {
+        User = who;
+    }
```

Comment:
```suggestion
    public MeleeSpeechConfigureActionMessage(EntityUid user)
    {
        User = user;
    }
```

Small change but it's better to rename this for consistency.